nextage_gazebo/test/gz.test

```diff
@@ -0,0 +1,10 @@
+<launch>
+  <arg name="GUI" default='false' />
```

Comment:
+1
Makes better sense to pull this argument as an `arg`, since you may want to locally run the test where you can open Gazebo GUI if you want to.